### PR TITLE
stackrox: perfscale acs on rox subdomain

### PR DESCRIPTION
**Pull request by davdhacs**
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/control-plane/openshift-qe-control-plane-chain.yaml
with
* stackrox cluster profile, and
* stackrox elasticsearch host, user, password
* scheduled nightly to match the openshift-qe-perfscale nightly run (on nightly ocp with stable acs)

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-payload-control-plane-6nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml [hunk 1]
```diff
@@ -54,6 +54,22 @@ tests:
     test:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws
+- always_run: false
```
[the remark this answers is not in the document]
Comment (jtaleric) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86.yaml]: if we want to enable `/test` we need to modify the stackrox config too, but we can do that on a follow on. 

@weekly is fine with me, much of this is based on the teams needs. The only recommendation I have is to try to match the PerfScale 24 node aws test to use that as a baseline.

File: ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh [hunk 2]
```diff
@@ -11,8 +11,18 @@ pushd /tmp
 python -m virtualenv ./venv_qe
 source ./venv_qe/bin/activate
 
-ES_PASSWORD=$(cat "/secret/password")
-ES_USERNAME=$(cat "/secret/username")
+ES_HOST=${ES_HOST:-"search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"}
```
[the remark this answers is not in the document]
Comment (tommartensen) [reply ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh:16]: Hm I don't think this domain should be in code...

**Review by jtaleric — APPROVED** [review]
/lgtm

**Pull request merged**